libasynql/src/poggit/libasynql/base/QueryRecvQueue.php

```diff
@@ -50,12 +50,12 @@ public function publishError(int $queryId, SqlError $error) : void{
 	}
 
 	public function fetchResults(&$queryId, &$results) : bool{
-		if($this->count() > 0){
-			while(($row = $this->shift()) === null);
-			[$queryId, $results] = unserialize($row, ["allowed_classes" => true]);
-			return true;
+		while(is_int($row = $this->shift())); //Somehow $this->availableThreads are here, so we have to remove them. TODO: Remove available threads in the next major update.
```

Comment:
You might as well remove it directly, it's not going to do what you want anyway.

Reply:
Hmm, I guess I'll deprecate the methods then.

Reply:
Seems to me like they are internal methods, no one should be using them.